src/pages/nfts/[address].tsx

```diff
@@ -165,6 +176,16 @@ export default function NftByAddress({
   const defaultListing = nft?.listings.find(
     (listing) => listing?.auctionHouse?.address.toString() === HOLAPLEX_MARKETPLACE_ADDRESS
   );
+
+  const otherListings = nft?.listings.filter(
+    (listing) => listing.auctionHouse?.address.toString() !== HOLAPLEX_MARKETPLACE_ADDRESS
+  );
+
+  const magicEdenListing = nft?.listings.find((listing) => listing.auctionHouse === null);
```

Comment:
Should check on the marketplace program instead of auctionHouse being null.

Reply:
oh woops i no longer use this in favor of the dynamic way